docs/language-reference.rst

```diff
@@ -1664,6 +1664,14 @@ Exploit only outer parallelism in the attributed SOAC.
 Do not perform any dynamic safety checks (such as bound checks) during
 execution of the attributed expression.
 
+``warn_on(safety_checks)``
```

Comment:
This should be `warn(safety_checks)`, right?

Reply:
Yes.